cluster-scope/base/clusterrolebindings/scc-privileged-opf-ci-pipelines-aicoe-ci/clusterrolebinding.yaml

```diff
@@ -0,0 +1,13 @@
+---
+apiVersion: rbac.authorization.k8s.io/v1beta1
+kind: ClusterRoleBinding
+metadata:
+  name: scc-privileged-opf-ci-pipelines-aicoe-ci
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: system:openshift:scc:privileged
+subjects:
+  - kind: ServiceAccount
+    name: aicoe-ci
+    namespace: opf-ci-pipeline
```

Comment:
did you mean `opf-ci-pipelines` (the `s` at the end) ?

Reply:
good catch, i missed that :0

Reply:
Hmm.. I think I meant precisely that. :smile: Thanks! :+1: